File: udemy/pascal/projetos/operacoes_mat.pas
program operacoes_mat;
var num1, num2, soma, mult, divisao, sub : real;
begin
     write('digite um numero: ');
     readln(num1);
     write('digite um outro numero: ');
     readln(num2);
     soma := num1+num2;
     divisao:= num1/num2;
     mult := num1*num2;
     sub := num1-num2;
     writeln('A soma de ',num1:6:2,' e ',num2:6:2,' eh: ',soma:6:2);
     writeln('A multiplicacao de ',num1:6:2, ' e ',num2:6:2,' eh: ',mult:6:2);
     writeln('A subtracao de ',num1:6:2,' e ',num2:6:2,' eh: ', sub:6:2);
     writeln('A divisao de ',num1:6:2, ' e ', num2:6:2,' eh: ',divisao:6:2);
     writeln('------------final do sistema------------');
     readln();
end.

File: udemy/java/operacoes/src/operacoes/operacoes_mat.java
package operacoes;
import javax.swing.JOptionPane;
public class operacoes_mat {
	public static void main(String[] args) {
	float	num1, num2, mult,div,sub, soma;
	num1 = 	Float.parseFloat(JOptionPane.showInputDialog("Digite o primeiro numero: "));
	num2 = Float.parseFloat(JOptionPane.showInputDialog("Digite o segundo valor"));
	mult = num1*num2;
	soma = num1+num2;
	sub = num1-num2;
	div = num1/num2;
	
	JOptionPane.showMessageDialog(null, "multiplicacao: "+mult);
	JOptionPane.showMessageDialog(null, "divisao: "+div);
	JOptionPane.showMessageDialog(null, "soma: "+soma);
	JOptionPane.showMessageDialog(null, "subtracao: "+sub);
	
	}
}
